workshop/1-Personalization/Lab-4-Evaluate-recommendations.ipynb

```diff
@@ -597,7 +805,7 @@
     "\n",
     "Now that you've enabled each personalization feature by setting the respective recommender and campaign ARN, you can test these personalization features through the Retail Demo Store's Web App UI. If you haven't already opened a browser window/tab to the Retail Demo Store Web UI, navigate to the CloudFormation console in this AWS account and check the Outputs section of the stack used to launch the Retail Demo Store. Make sure you're checking the base/root stack and not the nested stacks that were created. In the Outputs section look for the output named: WebURL and browse to the URL provided.\n",
     "\n",
-    "![CloudFormation Outputs](../images/cfn-webui-outputs.png)\n",
+    "![CloudFormation Outputs](https://raw.githubusercontent.com/aws-samples/retail-demo-store/master/workshop/images/cfn-webui-outputs.png)\n",
```

Comment:
I don't think we should link with a link that contains the branch.
why not keeping relative (../images...) reference?

Reply:
I changed the links since the images were no longer being displayed in the notebooks. They were all broken images. I'm not sure when that broke. Maybe when the notebook instance identifier was changed to al2.

Reply:
Taking another look at them to see if I can get them working another way.

Reply:
Now relative links are working again. Don't know why they were breaking before. Reverted to relative.